tests/phpunit/Visitor/ReflectionVisitorTest.php

```diff
@@ -316,6 +339,10 @@ private function parseAndTraverse(string $code, ?NodeVisitor $nodeVisitor = null
         $this->traverse(
             $nodes,
             [
+                new NameResolver(null, [
```

Comment:
Instead of having it repeated over, since it requires some special settings, what about having a static factory for it?